### PR TITLE
[MRG][cli] update modelci service logic

### LINKED ISSUE #202: CLI improvement
- [x] reorganize CLI 
- [x] show model zoo in the terminal with CLI

### PR DESCRIPTION
**Pull request by HuaizhengZhang**
Closes #202 

#### What has been done to verify that this works as intended?

Test under my server.

#### Does this change require updates to documentation?

has updated

`modelci service init` to start the service on a leader server

#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

File: modelci/cli/__init__.py [hunk 1]
```diff
@@ -1,57 +1,27 @@
...
-"""
-import click
-
-from modelci.app import (start as app_start, stop as app_stop)
-from modelci.cli import model_cli
-from modelci.utils import Logger
-from modelci.utils.docker_container_manager import DockerContainerManager
-
-logger = Logger(__name__, welcome=False)
+#  Copyright (c) NTU_CAP 2020. All Rights Reserved.
```
Comment (YuanmingLeee) [inline modelci/cli/__init__.py:1]: 2021

File: modelci/cli/__init__.py [hunk 2]
```diff
@@ -1,57 +1,27 @@
...
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
 
+import click
+from modelci.cli.modelci_service import *
 
 @click.group()
-@click.version_option()
 def cli():
-    """A complete platform for managing, converting, profiling, and deploying models as cloud services (MLaaS)"""
```
Comment (YuanmingLeee) [inline modelci/cli/__init__.py:21]: This line is for printing helper:
image

File: modelci/cli/__init__.py [hunk 3]
```diff
@@ -1,57 +1,27 @@
...
-    """Stop the ModelCI service and remove all containers."""
-    # remove all services
-    container_conn = DockerContainerManager()
-    container_conn.stop()
-    app_stop()
-    container_conn.remove_all()
-
-
-cli.add_command(model_cli.commands)
-cli.add_command(model_cli.models)
```
Comment (YuanmingLeee) [inline modelci/cli/__init__.py:54]: keep the `model_cli`, as I need this function for the model registration

Comment (HuaizhengZhang) [reply modelci/cli/__init__.py:54]: recovered

will improve this in the next PR

**Review by YuanmingLeee — APPROVED** [review]

**Pull request merged**